server/helpers.go

```diff
@@ -1,7 +1,46 @@
 package server
 
-import "net/http"
+import (
+	"net/http"
+	"path"
+)
 
 func location(w http.ResponseWriter, self string) {
 	w.Header().Add("Location", self)
 }
+
+// hsts add HTTP Strict Transport Security header with a max-age of two years
+// Inspired from https://blog.bracebin.com/achieving-perfect-ssl-labs-score-with-go
+func hsts(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add("Strict-Transport-Security", "max-age=63072000; includeSubDomains")
+		next.ServeHTTP(w, r)
+	})
+}
+
+// version handler adds X-Chronograf-Version header to responses
+func version(version string, h http.Handler) http.Handler {
+	fn := func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add("X-Chronograf-Version", version)
```

Comment:
official recommendation is not to add an `x-` prefix to custom headers. not sure if that conflicts with our conventions, or is worth worrying about, but that's at least the official recommendation of rfc6648

>Update 2: On June 2012, the deprecation of recommendation to use the "X-" prefix has become official as RFC 6648. Below are cites of relevance:

https://stackoverflow.com/a/3561399/92446

Reply:
Ya, I didn't want to break anything by changing the actual functions. these functions are relocated from other files with just them.